pkg/controller/node/rate_limited_queue.go

```diff
@@ -29,6 +29,7 @@ import (
 // TimedValue is a value that should be processed at a designated time.
 type TimedValue struct {
 	Value     string
+	UID       string
```

Comment:
The queue is supposed to be generic, this would be better as interface{} with casts as necessary in the controller.